build.gradle

```diff
@@ -66,6 +66,10 @@ subprojects {
     compileJava {
       sourceCompatibility = '1.8'
       targetCompatibility = '1.8'
+      //This flag is only supported and required when building with JDK11
+      if (JavaVersion.current() == JavaVersion.VERSION_11) {
```

Comment:
would checking for >8 be more future proof? what minimum java version supports --release ?

Reply:
Updated